tests/contrib/urllib3/test_urllib3.py

```diff
@@ -98,7 +99,7 @@ def test_args_kwargs(self):
 
         for args, kwargs in inputs:
 
-            with self.override_config("urllib3", {}):
+            with self.override_http_config("urllib3", {"_whitelist_headers": set()}):
```

Comment:
Not a big fan of the implementation dependent `_whitelist_headers` - probably something we should work into our test helpers at some point. Any change to this implementation detail could break these tests.

Reply:
yeah, definitely agree, this is not ideal, but if I don't have this then these tests leak their config changes since the previous `override_config` code did nothing.